packages/mdx/lib/plugin/recma-jsx-rewrite.js

```diff
@@ -356,13 +359,23 @@ export function recmaJsxRewrite(options = {}) {
             }
 
             for (const [
-              invalidComponentName,
-              variable
-            ] of invalidComponentNameToVariable) {
+              _componentsId,
+              _componentName
+            ] of idToInvalidComponentName) {
```

Comment:
I still kinda like `name`, `value` better, but I’m not blocking on this.
I feel a bit more strongly about not using `_` to start variable names, that is often used with variables (parameters) to mark them as explicitly not used

Reply:
Understood, removed those `_`s! I also tried using `name` and `value`, but noticed it leads to `{value: name}` and `{name: value}` in the sections where they're used. It... kinda made my head spin, so I chose `[id, componentName]` instead 😆